tests/regression/summarize_test_results.py

```diff
@@ -211,18 +211,45 @@ def save_file(result_data: dict, output_path: str, file_name: str):
     df.to_csv(os.path.join(output_path, file_name))
 
 
+def merge_dict(target, source, overwrite=False):
```

Comment:
It seems that if `target` is `{1:2}` and `source` is `{3:4}`, `merge_dict(target, source)` doesn't change `target`. Is it ok?

Reply:
yes, it will do. i think it's better to rename the function of it due to it would not cover the general `dict` merge operation. it's only worthy to the regression result `dict` generated with different train types or label types.